tests/framework/builder.py

```diff
@@ -121,7 +123,7 @@ def __init__(self, microvm):
         self._microvm = microvm
 
     def create(self,
-               disks: [DiskArtifact],
+               disks,
```

Comment:
why remove type hint ?

Reply:
because it's no longer a list of `Artifacts`, it's currently a list of disk local paths, but it can be a list of different objects really, no need to tie it to a specific type.